README.md

```diff
@@ -1,10 +1,33 @@
+- [Bluetail](#bluetail)
+  * [Interfaces](#interfaces)
+  * [Installation](#installation)
+    + [Running Bluetail locally (with Vagrant)](#running-bluetail-locally--with-vagrant-)
+    + [Running Bluetail locally (without Vagrant)](#running-bluetail-locally--without-vagrant-)
```

Comment:
These two "running bluetail locally…" links don‘t work. I think it’s because of the stray `-` at the end of each one?